app/_landing_pages/dedicated-cloud-gateways.yaml

```diff
@@ -147,6 +147,16 @@ rows:
               cta:
                 text: See supported regions
                 url: /konnect-platform/geos/
+      - blocks:
+          - type: card
+            config:
+              title: Dedicated Cloud Gateway domain breaking changes
+              description: |
+                Review domain breaking changes for Dedicated Cloud Gateways and migrate to the new domain before September 30, 2025.
+              icon: /assets/icons/domain.svg
+              cta:
+                text: Review breaking changes
+                url: /dedicated-cloud-gateways/breaking-changes/
```

Comment:
Rather than a tile, I'd actually want to see this as a banner near the top of the page (maybe a yellow `warning` callout?). I think as a tile, this gets lost and just looks like it's some optional thing for people.